python/tk_nuke/menu_generation.py

```diff
@@ -829,8 +824,11 @@ def get_documentation_url_str(self):
             doc_url = self.app.documentation_url
             # Deal with nuke's inability to handle unicode.
             if type(doc_url) == str:
-                doc_url = sgutils.ensure_str(
-                    unicodedata.normalize("NFKD", doc_url), "ascii", "ignore"
+                doc_url = unicodedata.normalize("NFKD", doc_url)
+                doc_url = (
+                    doc_url.decode("ascii", "ignore")
+                    if isinstance(doc_url, bytes)
+                    else str(doc_url)
```

Comment:
Given the history of this method and why it was created, I suggest to clean it up:


```suggestion
if not self.app:
  return

return self.app.documentation_url
```